skyportal/tests/api/test_groups.py

```diff
@@ -120,3 +120,59 @@ def test_public_group(view_only_token):
     assert status == 200
     assert response["status"] == "success"
     int(response["data"]["id"])
+
```

Comment:
Can you add the following tests?

   

- [ ] Grant a stream to a user, then delete the stream, and check that the user still exists, and has access to their original streams, minus the new one

- [ ]  Grant a stream to a group, then delete the stream, and check that the group still exists, and has access to its original streams, minus the new one

- [ ] Attach a filter to a group, then delete the group, and check that the filter has also been deleted in a cascade.

Reply:
Added